docs/tutorials/timeslices.md

```diff
@@ -0,0 +1,258 @@
+<!--
```

Comment:
should we mention somewhere that the first and last timeslice won't always be a full --slice-duration?

Reply:
Added a section for this.